python/lsst/obs/base/camera_tests.py

```diff
@@ -71,18 +72,31 @@ def setUp_camera(
             plate_scale=plate_scale,
         )
 
+    def _butler_args(self):
+        """Arguments to pass to butler get in addition to camera dataset type.
+
+        Returns
+        -------
+        kwargs : `dict`
+            The arguments to add.
+        """
+        kwargs = {}
+        if isinstance(self.butler, Butler):
```

Comment:
Multiple inheritance makes it all a bit murky as to whether it will find it. I felt it was clearer for this temporary fix to do the explicit test in this file.